Add more prop emotes

BackSH00TER opened the pull request:
Added the following general emote:
elbow
![image](https://github.com/user-attachments/assets/c6b72dc0-1040-41b3-9c4a-a5156e56d06f)

Added a handful of new prop emotes.

Added the following prop emotes:
medbag2
![image](https://github.com/user-attachments/assets/8f469b6b-f697-4c39-bf3a-8a14e1f13904)

doggy
![image](https://github.com/user-attachments/assets/0bd9951a-1b24-4369-bc11-d423fc7a5b90)

stickhummingbird
![image](https://github.com/user-attachments/assets/2f1aa886-9462-4c08-98e9-16f51d1559ee)

stickbutterfly
![image](https://github.com/user-attachments/assets/37dff601-3d1e-434d-ad42-309078b759b7)

vulture (fixed position from what is in the recording)
![image](https://github.com/user-attachments/assets/1d963bea-9c1d-4cb0-8514-72a99ef5fb97)

present
![image](https://github.com/user-attachments/assets/c701bd27-2063-4f74-ae0f-e62afe35df44)

vlog
![image](https://github.com/user-attachments/assets/af441d3f-cdd6-4d09-8805-44a147b8343d)

vlog2
![image](https://github.com/user-attachments/assets/f7f91dd9-edd8-4814-b68d-0658a7c2d517)

hoe
![image](https://github.com/user-attachments/assets/6130e3d5-42fa-44a2-be7d-2d67c0788ccb)

hoe2
![image](https://github.com/user-attachments/assets/e08f1046-34fd-47a2-82ad-509f0c2c7a84)

hoe3
![image](https://github.com/user-attachments/assets/d89717bb-ac5a-4041-b6f2-172da9587d72)

hoe4
![image](https://github.com/user-attachments/assets/5f9391e7-55dc-40ff-bb7c-759de9038de8)

hammer
![image](https://github.com/user-attachments/assets/c0f56e73-c318-40e1-955d-a12358210db7)

mallet
![image](https://github.com/user-attachments/assets/0bb26476-2054-48e0-8f6f-4f6921007b7c)

drilltool
![image](https://github.com/user-attachments/assets/1bc5d52e-d517-4e43-a2a3-032f577e817a)

nailgun
![image](https://github.com/user-attachments/assets/b9e5ebf0-b179-4237-aa0e-2ea89d4d875a)

jackhammer
![image](https://github.com/user-attachments/assets/8f777d2e-7983-49f7-a7b2-fb0025070f1e)

circularsaw
![image](https://github.com/user-attachments/assets/2c8529d2-fd77-446b-ab2c-0f390804385c)

syringe
![image](https://github.com/user-attachments/assets/107c470c-9535-481b-a4e9-f8b66b1e879f)

cranialsaw
![image](https://github.com/user-attachments/assets/e87f3203-90d4-450d-98b4-3b43628f6310)

scalpel
![image](https://github.com/user-attachments/assets/9f551ab6-aa52-4239-a2af-e430dfb00bc6)

bonesaw
![image](https://github.com/user-attachments/assets/7da73182-b5e1-4910-91fb-fc7dcff01971)


These can be previewed in this recording:
https://github.com/user-attachments/assets/82c750b2-dd4b-4efc-81e0-af8ff1247d98

Ended up adding more after making the recording:
protest4
![image](https://github.com/user-attachments/assets/cd09f783-ae84-4886-bc9e-abf7f5538714)

protest5
![image](https://github.com/user-attachments/assets/b5805688-0585-4474-a1c1-a373853cd1e8)

protest6
![image](https://github.com/user-attachments/assets/498c9ea4-fcac-46db-b41c-d65cdabce059)

ssign10 (Speed limit 15)
![image](https://github.com/user-attachments/assets/7950134c-ad60-4dc3-9a6e-619378807ddf)

ssign11 (Speed limit 35)
![image](https://github.com/user-attachments/assets/05b15d41-e643-485c-b4c9-847c23fde3af)

ssign12 (Speed limit 60)
![image](https://github.com/user-attachments/assets/f38a495d-5d0d-4ed7-afc7-fb80fc810659)

suitcase3
![image](https://github.com/user-attachments/assets/455816a9-a666-4e7f-b491-92287363329b)

champw2
![image](https://github.com/user-attachments/assets/7bab0dbb-892a-4d4b-b0a7-b73f99315dc1)

champw3
![image](https://github.com/user-attachments/assets/62fc221f-c918-4d27-ba9d-0c0edce596f7)

handbag
![image](https://github.com/user-attachments/assets/888f5102-c5c6-438e-a046-0de00edc96b8)

handbag2
![image](https://github.com/user-attachments/assets/2a144abe-8372-4574-abf2-120e0a9e8709)

handbag3
![image](https://github.com/user-attachments/assets/1dac6869-29a1-4e0d-880e-be766f18a34f)

beachbag
![image](https://github.com/user-attachments/assets/cabe1ca5-7f2f-44c4-947d-f3854e7cffeb)

beachbag2
![image](https://github.com/user-attachments/assets/dcba4ed4-0c0d-4fc0-941f-23e539a07f51)

beachbag3
![image](https://github.com/user-attachments/assets/49ea405e-931c-493b-81d7-5749a951690d)

dufbag2
![image](https://github.com/user-attachments/assets/a500c433-5d19-47e0-8e93-ae47bf773e50)

dufbag3
![image](https://github.com/user-attachments/assets/04daa243-ac64-4c28-a7bd-4132734b752a)

Scullyy commented:
Hello, thank you for making a PR. Before merging though could you:

- Adjust the elbow emotes location so that it's in alphabetical order based on the command.
- Adjust the prop emotes so that they're in alphabetical order in the list based on the commands.
- Resolve the issues with the floating hammer in the left hand.
- Adjust the position of the vulture so that it doesn't have your arm out when it's on the shoulder, maybe just place it on the arm as the emote would be more fitting for like, I'll post an example image below :)

https://farm1.staticflickr.com/148/393744656_d1711f39c9_z.jpg

BackSH00TER commented:
> Hello, thank you for making a PR. Before merging though could you:
> 
> * Adjust the elbow emotes location so that it's in alphabetical order based on the command.
> * Adjust the prop emotes so that they're in alphabetical order in the list based on the commands.
> * Resolve the issues with the floating hammer in the left hand.
> * Adjust the position of the vulture so that it doesn't have your arm out when it's on the shoulder, maybe just place it on the arm as the emote would be more fitting for like, I'll post an example image below :)
> 
> https://farm1.staticflickr.com/148/393744656_d1711f39c9_z.jpg

Sure no problem. 

I believe floating hammer was just the prop got stuck at some point, I'll double check.

@Scullyy I have a handful more I'm planning to add as well. Would you prefer I split these up in separate PRs or keep adding them onto here?

BackSH00TER commented:
Adjusted vulture onto the arm
![image](https://github.com/user-attachments/assets/ec25efc5-69a0-4073-a4f9-1075996824eb)

And sorted the additions alphabetically

BackSH00TER commented:
Okay I think I added all the ones I was planning to add for now ;) 

I had added these a long time ago when we were using dpemotes, figured now that we're switching to scullyemotes, might as well contribute these up for everyone else to enjoy as well. :)

Scullyy commented:
> Okay I think I added all the ones I was planning to add for now ;)
> 
> I had added these a long time ago when we were using dpemotes, figured now that we're switching to scullyemotes, might as well contribute these up for everyone else to enjoy as well. :)

Thank you again for your contributions, I'll merge these now :)
